admin-ui/src/pages/models-page.tsx

```
@@ -0,0 +1,514 @@
+type SortDir = "asc" | "desc"
+
+type SortKey =
+  | "name"
+  | "endpoints"
+  | "originalId"
+  | "context"
+  | "features"
+  | "multiplier"
+
+function fmtTokensCompact(n?: number): string {
+  if (n == null) return ""
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)